azure-pipelines/release.yml

```diff
@@ -2,7 +2,7 @@ steps:
 - task: AzureKeyVault@1
   displayName: 'Azure Key Vault: ado-secrets'
   inputs:
-    azureSubscription: 'ClientToolsInfra_670062 (88d5392f-a34f-4769-b405-f597fc533613)'
+    azureSubscription: 'STS Pipelines'
     KeyVaultName: 'ado-secrets'
     SecretsFilter: 'github-distro-mixin-password,ado-crossplatbuildscripts-password'
```

Comment:
I don't think the `github-distro-mixin-password` is needed, since I couldn't find any references here and in the sqltoolsserviceRelease.ps1 script in CrossPlatBuildScripts that gets called. The mixin step is only done in ADS, so it's also weird if a secret with 'mixin' in the name would be needed here. 

After I make sure the pipelines start passing again with this current change, I'll try removing it and see if anything breaks.